webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -10,7 +11,7 @@ import {
 } from 'types';
 import { numericSorter } from 'utils/data';
 import { formatDatetime } from 'utils/date';
-import { humanReadableBytes, humanReadableFloat } from 'utils/string';
+import { humanReadableBytes } from 'utils/string';
```

Comment:
**question:** The component looks good! I think this is right, by moving away from the `utils/string` and completely into the component. I don't know 100% but I don't think we will have a need to run `humanReadableFloat` outside of the component since this is directly related to presentation...